features/product/viewing_products/viewing_products_from_specific_taxon.feature

```diff
@@ -22,3 +22,8 @@ Feature: Viewing products from a specific taxon
     Scenario: Viewing information about empty list of products from a given taxon
         When I browse products from taxon "Sad"
         Then I should see empty list of products
+
+    @api
+    Scenario: Searching products with non existing taxon
```

Comment:
Let's add a scenario with coverage of the product without root taxon assigned. Right now it will be shown, but the list should still be empty